aten/src/ATen/cpu/vec/vec512/vec512_float8.h

```diff
@@ -392,6 +503,65 @@ inline Vectorized<Float8_e4m3fn> Vectorized<Float8_e4m3fn>::le(const Vectorized<
   return (*this <= other) & Vectorized<Float8_e4m3fn>(1.0f);
 }
 
+template <>
+class Vectorized<Float8_e5m2>: public Vectorizedf8<Float8_e5m2> {
+public:
+  using Vectorizedf8::Vectorizedf8;
+
+  using value_type = Float8_e5m2;
+
+  Vectorized<Float8_e5m2> eq(const Vectorized<Float8_e5m2>& other) const;
+  Vectorized<Float8_e5m2> ne(const Vectorized<Float8_e5m2>& other) const;
+  Vectorized<Float8_e5m2> gt(const Vectorized<Float8_e5m2>& other) const;
+  Vectorized<Float8_e5m2> ge(const Vectorized<Float8_e5m2>& other) const;
+  Vectorized<Float8_e5m2> lt(const Vectorized<Float8_e5m2>& other) const;
+  Vectorized<Float8_e5m2> le(const Vectorized<Float8_e5m2>& other) const;
+};
+
+Vectorized<Float8_e5m2> inline operator+(const Vectorized<Float8_e5m2>& a, const Vectorized<Float8_e5m2>& b) {
```

Comment:
We plan to remove support for `+, -, *, /` from float8 dtypes in PyTorch, it would be great to not add more callsites here. If you need this to make the compiler happy, would be great to add a comment saying that this is only here for the compiler and should be deleted in the future.

Reply:
Thanks and yes, it's added to make compiler happy. There are some kernel dispatch https://github.com/pytorch/pytorch/blob/edc2d539d1e8040065b6a0f83638218e16d44283/aten/src/ATen/native/cpu/BinaryOpsKernel.cpp#L157 which includes FP8 data types. I will add the comment accordingly.

Reply:
Comments added here and also in PR https://github.com/pytorch/pytorch/pull/152417, please kindly take a look again.